src/main/java/com/notably/logic/parser/NewCommandParser.java

```diff
@@ -61,6 +68,7 @@ public NewCommandParser(Model notablyModel) {
 
         AbsolutePath path = ParserUtil.createAbsolutePath(title, notablyModel.getCurrentlyOpenPath());
         commands.add(new OpenCommand(path));
+        logger.info("Command jumps to the block after creation");
```

Comment:
I feel like there should be a `logger.info` after each `commands.add` in this `CommandParser`? Since you're creating two commands and letting the caller execute them. For that, perhaps you should add one `logger.info` above after `commands.add(new NewCommand(block))`?

Also, I think we should rephrase this `logger.info` to better convey your message. How about phrasing it as `OpenCommand with the path '%s' is created`?

Reply:
Added logger.info for command.add !